lnwire/features.go

```diff
@@ -489,6 +489,10 @@ func (fv *FeatureVector) Features() map[FeatureBit]struct{} {
 // Clone copies a feature vector, carrying over its feature bits. The feature
 // names are not copied.
 func (fv *FeatureVector) Clone() *FeatureVector {
-	features := fv.RawFeatureVector.Clone()
+	var features *RawFeatureVector
+
+	if fv.RawFeatureVector != nil {
```

Comment:
👍 

Should likely tack on a simple test case just to clamp this down further.

Reply:
Dropped this commit, I don't think it is necessary anymore. Also, `EmptyFeatureVector()` doesn't create nil raw features.